keps/sig-apps/4212-declarative-node-maintenance/kep.yaml

```
@@ -0,0 +1,57 @@
+title: Declarative Node Maintenance
+kep-number: 4212
+authors:
+  - "@atiratree"
+owning-sig: sig-apps
+participating-sigs:
+  - sig-apps
+  - sig-autoscaling
+  - sig-cli
+  - sig-node
+  - sig-scheduling
+status: provisional
+creation-date: 2023-09-15
+reviewers:
+  - "@sftim"
+  - "@fabiand"
+  - "@mmerkes"
+  - "@rptaylor"
+  - "@pwschuurman"
+  - "@dchen1107"
+  - "@soltysh"
+  - "@dbenque"
+  - "@wangzhen127"
+  - "@alvaroaleman"
+  - "@razo7"
+  - "@adammw"
```

Adding myself, plus keeping the list sorted.

```suggestion
reviewers:
  - "@adammw"
  - "@alvaroaleman"
  - "@dbenque"
  - "@dchen1107"
  - "@fabiand"
  - "@kwilczynski"
  - "@mmerkes"
  - "@pwschuurman"
  - "@razo7"
  - "@rptaylor"
  - "@sftim"
  - "@soltysh"
  - "@wangzhen127"
```